scripts/west_commands/runners/linkserver.py

```diff
@@ -150,7 +150,10 @@ def do_run(self, command, **kwargs):
                            ['-ex', f'target remote {self.gdb_host}:{self.gdb_port}'])
 
                 if command == 'debug':
-                    gdb_cmd += [ '-ex', 'load', '-ex', 'monitor reset']
+                    # If the flash node points to ram, linserver treats
+                    # the ram as inaccessible and does not flash.
+                    gdb_cmd += ['-ex', 'set mem inaccessible-by-default off']
+                    gdb_cmd += ['-ex', 'monitor reset', '-ex', 'load']
```

Comment:
why is the order of load, reset being reversed to reset, load?

Reply:
If we are going to reset, we would want it to be before a load.

Reply:
But I do see a typo I will fix in the next push if I don't get further comments